api/v1alpha1/driver_types.go

```diff
@@ -77,7 +77,7 @@ type PluginResourcesSpec struct {
 	Plugin    *corev1.ResourceRequirements `json:"plugin,omitempty"`
 }
 
-type PluginSpec struct {
+type NodePlugin struct {
```

Comment:
can you also update PluginResourcesSpec to NodePluginResourceSpec as well and same for ProvisionerResourcesSpec  to ControllerPluginResourceSpec?

Reply:
done